src/status_im/contexts/wallet/home/view.cljs

```diff
@@ -60,7 +60,7 @@
            :data                              cards
            :horizontal                        true
            :separator                         [rn/view {:style style/separator}]
-           :render-fn                         quo/account-card
+           :render-fn                         (fn [item] [quo/account-card item])
```

Comment:
why is this needed?

Reply:
It was throwing malli errors before this. I don't know why 🤔

Reply:
@ilmotta The original flat-list implementation expects a function to be passed.

Just saying again that we should revisit our wrapper :eye:

Reply:
But `quo/account-card` is a function, so it surprises me that we need to wrap it in a dummy closure.

Reply:
> Just saying again that we should revisit our wrapper 👁️

Just saying... I see you! Yes, that wrapper function seems very inefficient with the creation of anonymous functions over and over.